hr_contract_type/models/hr_contract_type.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2021 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ContractType(models.Model):
+    _name = "hr.contract.type"
+    _description = "Contract Type"
+
+    name = fields.Char(required=True)
```

Comment:
Hi thanks for the pr @mtelahun,
Migration LGTM.
I have a general question about this module:

I don't understand why there is no sql_constraint to avoid duplication of name.